tests/integration/yamls/cluster.yaml

```diff
@@ -2,3 +2,18 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  kubeadmConfigPatches:
```

Comment:
Any links to `kind` (or not) docs that treat about this?

Reply:
Kind docs on customizing kubeadm: https://kind.sigs.k8s.io/docs/user/configuration/#kubeadm-config-patches
And kubeadm docs on configuring control plane components: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/control-plane-flags/

You reckon I should put those in the comment?

Reply:
yes, please